src/richie/apps/core/templates/richie/base.html

```diff
@@ -54,6 +54,23 @@
         {% endblock meta %}
         {% endspaceless %}
 
+        {% block google_analytics %}
```

Comment:
Remove "google" from the block name so it can be overridden with another analytics platform?  Maybe review this whole PR with override in mind?

Reply:
Rename it, from "google analytics" to "web tracking" ?

Reply:
Or web_analytics?